test/e2e/main_test.go

```diff
@@ -50,7 +50,21 @@ func main(m *testing.M) int {
 		defer cleanup()
 	}
 
-	return m.Run()
+	exitCode := m.Run()
+
+	tests := []testing.InternalTest{{
+		Name: "NoReconcilationErrors",
+		F: func(t *testing.T) {
+			f.AssertNoReconcileErrors(t)
+		},
```

Comment:
Wouldn't this also work?

```suggestion
		F: f.AssertNoReconcileErrors,
```

Reply:
aha! good catch :D